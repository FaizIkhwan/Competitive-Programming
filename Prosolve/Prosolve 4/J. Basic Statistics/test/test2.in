47 85 -77 -42 -68 -78 -96 63 -42
