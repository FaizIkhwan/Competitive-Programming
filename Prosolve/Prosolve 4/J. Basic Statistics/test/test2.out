Mean : -23.11
Median : -68.00
